addon/components/frost-chart-svg-plot-y-grid.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * Component definition for the frost-chart-svg-plot-y-grid component
+ */
+import Ember from 'ember'
+const {get} = Ember
+import computed, {readOnly} from 'ember-computed-decorators'
+import {isDomainValid} from 'ember-frost-chart/utils/validation'
+import {Component} from 'ember-frost-core'
+import {PropTypes} from 'ember-prop-types'
+
+import layout from '../templates/components/frost-chart-svg-plot-y-grid'
+
+export default Component.extend({
+  // == Dependencies ==========================================================
+
+  // == Keyword Properties ====================================================
+
+  layout,
+  tagName: 'g',
+
+  // == PropTypes =============================================================
+
+  /**
+   * Properties for this component. Options are expected to be (potentially)
+   * passed in to the component. State properties are *not* expected to be
+   * passed in/overwritten.
+   */
```

Comment:
When I finally get around to moving the blueprints out of core, I'll add a `--pro` option to the generators to leave off the explanatory comments in the generated files, so newbies can still get the hints, but the rest of us don't have to delete them manually.